### PR TITLE
Proposition: use conditional probability when comparing gen-disc pair

### PR DESCRIPTION
**Pull request by Jirixek**
I think it makes more sense to use conditional instead of joint probability in this case as the linearness (of the conditional probability) was shown earlier on the slides and it makes comparison clearer.

### CODE REVIEW

**Comment by foxik** [comment]
You are right, it makes sense to use conditional probability in both. I was originally careful because the NB models the $$\log p(C_k|x)$$ only up to a constant; but the same is true in fact also for the multiclass logistic regression.

So I rewrote the formulation to state that both models model logits (i.e., $$\log p(C_k|x) + c$$) as a linear model (sorry for so many models in the sentence ;-)).

**Comment by foxik** [comment]
Of course, you get a community-work point for the suggestion :+1:
